src/main/java/com/esri/samples/graphics_update/client_app/MoveGraphicsSample.java

```diff
@@ -0,0 +1,200 @@
+        uniqueValueRenderer.setDefaultSymbol(simpleMarkerSymbol);
+
+        // all of the potential status values for the vehicles
+        ArrayList<String> symbolNames = new ArrayList<>(Arrays.asList("Available", "Off duty", "On route", "Attending call"));
+
+        // loop through the symbols to create the unique values for each status
+        for (String symbolName : symbolNames) {
+            ListenableFuture<Symbol> searchResult = vehicleStyle.getSymbolAsync(Collections.singletonList(symbolName));
+            searchResult.addDoneListener(()-> {
+                try {
+                    Symbol symbol = searchResult.get();
+
+                    switch (symbolName) {
+                        case "Available" :
+                            UniqueValueRenderer.UniqueValue uniqueAvailableValue =
+                                    new UniqueValueRenderer.UniqueValue(symbolName, symbolName, symbol, Collections.singletonList("AVAILABLE"));
+                            uniqueValueRenderer.getUniqueValues().add(uniqueAvailableValue);
+                            break;
+                        case "Off duty" :
+                            List<Object> offDutyValue = new ArrayList<>();
+                            offDutyValue.add("OFF_DUTY");
+                            UniqueValueRenderer.UniqueValue uniqueOffDutyValue =
+                                    new UniqueValueRenderer.UniqueValue(symbolName, symbolName, symbol, Collections.singletonList("OFF_DUTY"));
+                            uniqueValueRenderer.getUniqueValues().add(uniqueOffDutyValue);
+                            break;
+                        case "On route" :
+                            List<Object> onRouteValue = new ArrayList<>();
+                            onRouteValue.add("ON_ROUTE");
+                            UniqueValueRenderer.UniqueValue uniqueOnRouteValue =
+                                    new UniqueValueRenderer.UniqueValue(symbolName, symbolName, symbol, Collections.singletonList("ON_ROUTE"));
+                            uniqueValueRenderer.getUniqueValues().add(uniqueOnRouteValue);
+                            break;
+                        case "Attending call":
+                            List<Object> attendingCallValue = new ArrayList<>();
+                            attendingCallValue.add("ATTENDING_CALL");
+                            UniqueValueRenderer.UniqueValue uniqueAttendingCallValue =
+                                    new UniqueValueRenderer.UniqueValue(symbolName, symbolName, symbol, Collections.singletonList("ATTENDING_CALL"));
+                            uniqueValueRenderer.getUniqueValues().add(uniqueAttendingCallValue);
+                            break;
+                        default:
+                    }
+                } catch (InterruptedException e) {
+                    e.printStackTrace();
+                } catch (ExecutionException e) {
+                    e.printStackTrace();
```

Comment:
Make one `catch` block with just `Exception` / `InterruptedException | ExecustionException` ? Unless there's a specific reason for this!